app/build.gradle

```
@@ -21,7 +21,7 @@ android {
         versionCode 1
         versionName "1.0"
 
-        buildConfigField "String", "BASE_URL", properties["base_url"]
+        buildConfigField "String", "BASE_URL", properties["base.url"]
```

local properties에 base_url로 써주시고 다시 수정 부탁드릴게요!
```suggestion
        buildConfigField "String", "BASE_URL", properties["base_url"]
```

---

넹~